src/compiler/scala/tools/nsc/typechecker/TypeDiagnostics.scala

```diff
@@ -746,7 +746,7 @@ trait TypeDiagnostics {
     }
     def apply(unit: CompilationUnit): Unit = if (warningsEnabled && !unit.isJava && !typer.context.reporter.hasErrors) {
       val body = unit.body
-      // TODO the message should distinguish whether the unusage is before or after macro expansion.
+      // TODO the message should distinguish whether the usage is before or after macro expansion.
```

Comment:
This was not an error. It is detecting whether a symbol is unused.

Reply:
Non-usage

Reply:
I still have the page open from the other review: https://en.wikipedia.org/wiki/English_words_first_attested_in_Chaucer

Reply:
I'm not invested either way, but clearly mine is funnier.

The TODO should be a ticket and not a comment anyway.